### ISSUE TITLE
workflow for programme responsiveness erroring at first line

### ISSUE THREAD

**Issue by ernestguevarra** [OxfordIHTM/ihtm-hackathon-2024#60]
This line of the `sudan_health_nutrition_5.R` script is erroring:

```R
#Create Time Column
cmam$time = match(cmam$Month,month.abb + (cmam$Year-2016)*12)
```

error is:

```R
Error in month.abb + (cmam$Year - 2016) * 12 : 
  non-numeric argument to binary operator
```

**Comment by marianortt** [comment]
Done, sent a pull request.

There was a problem with parenthesis 

cmam$time = match(cmam$Month,month.abb) + (cmam$Year-2016)*12

**Comment by ernestguevarra** [comment]
fixed by #61
